commit 203a30f05ed6e74c6fa8c6136dff152f26d43e93
Author: Narrat
Date:   Sun Jan 7 18:26:36 2018 +0100

    Enable dunstify
    
    Can more stuff than notify-send (like IDs) which I wanna have for sending notifications

diff --git a/trunk/PKGBUILD b/trunk/PKGBUILD
index 9b1ff1b..1f0eadc 100644
--- a/trunk/PKGBUILD
+++ b/trunk/PKGBUILD
@@ -21,11 +21,14 @@ md5sums=('f095a62ae24ec9de87596cadad19811a')
 
 build() {
 	make -C "$srcdir/$pkgname-$pkgver" PREFIX=/usr X11INC=/usr/include/X11 X11LIB=/usr/lib/X11
+	make -C "$srcdir/$pkgname-$pkgver" PREFIX=/usr X11INC=/usr/include/X11 X11LIB=/usr/lib/X11 dunstify
 }
 
 package() {
 	make -C "$pkgname-$pkgver" DESTDIR="${pkgdir}" PREFIX=/usr install
 	install -Dm644 "${srcdir}/$pkgname-$pkgver/LICENSE" "${pkgdir}/usr/share/licenses/${pkgname}/LICENSE"
+
+	install -Dm755 "${srcdir}/$pkgname-$pkgver/dunstify" "${pkgdir}/usr/bin/"
 }
 
 # vim:set ts=2 sw=2 et:
